ONSdigital/eq-terraform
# Enable dev mode during terraform apply for dev and preprod accounts

**_What**_
Allow the user to turn on dev mode via terraform environment variables. The default value is false.

**How to test**
1. Add `dev_mode="True"` to your terraform.tfvars and deploy a new environment.
2. Check that dev mode is enabled
3. Destroy that environment, remove the dev_mode variable from your terraform.tfvars and redeploy
4. Check that dev mode is not enabled

**Who can review**
Anyone apart from @warrenbailey
